{
 "bad": [
  "InjectorException",
  "async_func",
  "binding",
  "inject.__init__.BinderCallable",
  "inject.__init__.Binding",
  "inject.__init__.Constructor",
  "inject.__init__.ConstructorTypeError",
  "inject.__init__.Injectable",
  "inject.__init__.Injector",
  "inject.__init__.InjectorException",
  "inject.__init__.Provider",
  "inject.__init__.T",
  "inject.__init__._AttributeInjection",
  "inject.__init__._BINDING_LOCK",
  "inject.__init__._ConstructorBinding",
  "inject.__init__._INJECTOR",
  "inject.__init__._INJECTOR_LOCK",
  "inject.__init__._NEW_TYPING",
  "inject.__init__._ParameterInjection",
  "inject.__init__._ParametersInjection",
  "inject.__init__._RETURN",
  "inject.__init__._is_union_type",
  "inject.__init__._unwrap_union_arg",
  "inject.__init__.clear",
  "inject.__init__.configure",
  "inject.__init__.get_injector_or_die",
  "inject.__init__.instance",
  "inject.__init__.logger",
  "instance",
  "sync_func",
  "sys",
  "sys.version_info",
  "sys.version_info.major",
  "types.items",
  "typing.ForwardRef",
  "typing._GenericAlias",
  "wrapper"
 ],
 "deps": {
  "InjectorException": [],
  "async_func": [],
  "binding": [],
  "functools": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "inject": [],
  "inspect": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "instance": [],
  "logging": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "sync_func": [],
  "sys": [],
  "threading": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "types": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "typing": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "wrapper": []
 }
}